windowsclient.adoc

```diff
@@ -7,7 +7,7 @@ automated as possible.
 == Client Build VM ==
 
 First, install link:https://www.vagrantup.com/[Vagrant],
-link:https://packer.io/[Packer], and VirtualBox.  See
+link:https://github.com/mitchellh/packer/[Packer] (ensure you download and compile version 0.8.3 at min to avoid WinRM issues), and VirtualBox.  See
```

Comment:
I think the original link is a better place to point people to. They may move that github repo out of that personal directory at some stage. Download is one click away from the packer.io page.